calour/filtering.py

```diff
@@ -326,7 +328,7 @@ def _mean_abundance(data, axis, cutoff=0.01, strict=False):
     return res
 
 
-def _prevalence(x, cutoff=1/10000, fraction=0.5):
+def _prevalence(x, cutoff=1/10000, fraction=0.1):
```

Comment:
i like the default to be 0.5, so it keeps features present in at least 0.5 the samples.... but i see the logic in your suggestion... maybe remove the default and make it mandatory?

Reply:
let's make "fraction" mandatory (can move it before "cutoff")?

Reply:
i disagree. the previous 0.5 is too aggressive. what reason to make it mandatory?

Reply:
since we see people expect different values as default :)